job-runner/chain/traverser.go

```diff
@@ -349,35 +349,25 @@ func (t *traverser) Stop() error {
 	return err
 }
 
-// Status returns the status of currently running jobs in the chain.
-func (t *traverser) Status() (proto.JobChainStatus, error) {
-	t.logger.Infof("getting the status of all running jobs")
-
-	activeRunners, err := t.runnerRepo.Items()
-	if err != nil {
-		return proto.JobChainStatus{}, err
-	}
-
-	runningJobs := t.chain.Running()
-	status := make([]proto.JobStatus, len(runningJobs))
-	i := 0
-	for jobId, jobStatus := range runningJobs {
-		runner := activeRunners[jobId]
-		if runner == nil {
-			// The job finished between the call to chain.Running() and now,
-			// so it's runner no longer exists in the runner.Repo.
-			jobStatus.Status = "(finished)"
-		} else {
-			jobStatus.Status = runner.Status()
+func (t *traverser) Running() []proto.JobStatus {
+	runners := t.runnerRepo.Items()                       // map[string]interface{} keyed on jobId
```

Comment:
Items() returns map[string]Runner, not map[string]interface{}

Reply:
Thanks. Fixed.